community/coreutils/build

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
+
+export FORCE_UNSAFE_CONFIGURE=1
+
+./configure \
+    --prefix=/opt/gnu/coreutils \
+    --program-prefix=g \
+    --program-transform-name=s/stat/nustat/
```

Comment:
`nustat` -> `gnustat`? Or even `gstat`(?). I'm not sure how the name of this is typically handled when it conflicts. The BSDs might be a good place to look.

Reply:
> `nustat` -> `gnustat`? Or even `gstat`(?). I'm not sure how the name of this is typically handled when it conflicts. The BSDs might be a good place to look.

I  copy and pasted it from the freebsd make.conf actually, I'm sure they have a good reason for it.

Reply:
Sounds good to me then. :+1: